docs/openapi/components/schemas/common/GAPInspection.yml

```diff
@@ -0,0 +1,427 @@
+$linkedData:
+  term: GAPInspection
+  '@id': https://w3id.org/traceability#GAPInspection
+title: GAP Inspection
+description: A harmonized GAP (Good Agricultural Practices) or GAP Plus+ inspection report.
```

Comment:
Is "GAP Plus+" correct? It seems that should be either "GAP Plus" or "GAP+" rather than effectively being "GAP Plus Plus"...

Reply:
Yes, "GAP Plus+" is correct